api/config/config_local.json

```diff
@@ -108,7 +108,7 @@
   "featureFlags": {
     "unsafeAllowDeleteUser": true,
     "unsafeAllowAccessToAllTiersForRegisteredUsers": true,
-    "enableBillingUpgrade": true,
+    "enableBillingUpgrade": false,
```

Comment:
FYI: As I am activity working on this. I might turn local & test on soon. (after I merge my current backend PR)

Reply:
OK, sounds good. Just want to ensure staging is at least off unless we're launching imminently. I'll turn all of these off for now and you can flip back on when you're ready.

I think one thing that should be fixed is the initial scope configuration before we turn this back on: https://github.com/all-of-us/workbench/blob/master/ui/src/app/services/sign-in.service.ts#L105